exp-models/addon/models/experiment.js

```diff
@@ -24,6 +24,15 @@ export default DS.Model.extend(JamModel, {
     endDate: DS.attr('date'),
     structure: DS.attr(),
 
+    thumbnailId: DS.attr('string'),
```

Comment:
Design question: this field seems to impose a 1:1 relationship between experiments and thumbnails.

Given that, can you help clarify the reason we choose to store thumbnails in a separate collection, rather than as part of the experiment?